web/client/themes/default/less/timeline.less

```diff
@@ -242,6 +245,12 @@
                                 .border-color-var(@theme-vars[timeline-histogram-border-color]);
                             }
                             &.vis-range {
+                                &.interval {
+                                    .background-color-var(@theme-vars[timeline-histogram-border-color]);
+                                    .border-color-var(@theme-vars[main-bg]);
+                                    border-style: solid 5px;
+                                    border-radius: 5px;
```

Comment:
these changes should not be included in the 
// **************
// Theme
// **************
part

these are more layout changes
please move them in the 
// **************
// Layout section
// **************